types/mdx/react-tests.tsx

```diff
@@ -11,6 +12,14 @@ import MyRonComponent from "./MyComponent.ron";
 
 // Test setup — User code
 
+declare module "mdx/types" {
+    namespace JSX {
+        type Element = runtime.JSX.Element;
+        type ElementClass = runtime.JSX.ElementClass;
+        type IntrinsicElements = runtime.JSX.IntrinsicElements;
+    }
+}
```

Comment:
This looks like a duplicate of the changes declined in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69142 ?
The same concerns remain.

Reply:
@eps1lon to be clear, will MDX require changes to upgrade to React 19 types?

There's an issue over here, in case it would make sense to drop a suggestion in there:

- https://github.com/mdx-js/mdx/issues/2487

Reply:
Continuing over there.

Reply:
Yes, it’s only on a type level. It’s not just related to React, but to all popular JSX frameworks. We need to document this in MDX. The solution you see here works, but ideally I would like a cleaner workaround.